test/edu-benefits/1995/00-required.e2e.spec.js

```diff
@@ -0,0 +1,102 @@
+
+      // Service periods page.
+      client
+        .waitForElementVisible('label[for="root_view:newService"]', Timeouts.slow);
+      EduHelpers.completeServicePeriods(client, EduHelpers.testData.servicePeriods.data, true);
+      client.click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/military-history/service-periods');
+
+      // Military service page.
+      client
+        .waitForElementVisible('label[for="root_view:hasServiceBefore1978"]', Timeouts.slow);
+      // Another mysteriously required pause. If we don't wait here, then the click below will
+      // do nothing sometimes.
+      client.pause(1000);
+      EduHelpers.completeMilitaryService(client, EduHelpers.testData.militaryHistory.data, true);
+      client.click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/military-history/military-service');
+
+      // New school
+      client
+        .waitForElementVisible('label[for="root_educationType"]', Timeouts.slow);
+      EduHelpers.completeNewSchool(client, EduHelpers.testData.newSchool.data, true);
+      client.click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/school-selection/new-school');
+
+      // Old school
+      client
+        .waitForElementVisible('label[for="root_oldSchool_name"]', Timeouts.slow);
+      EduHelpers.completeOldSchool(client, EduHelpers.testData.oldSchool.data, true);
+      client.click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/school-selection/old-school');
+
+      // Contact information page.
+      client
+        .waitForElementVisible('label[for="root_preferredContactMethod"]', Timeouts.slow);
+      EduHelpers.completeContactInformation(client, EduHelpers.testData.contactInformation.data, true);
+      client.click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/personal-information/contact-information');
+
+      // Direct deposit page
+      client
+        .waitForElementVisible('label[for="root_bankAccountChange"]', Timeouts.slow);
+      EduHelpers.completeDirectDeposit(client, EduHelpers.testData.directDeposit.data, true);
+      client.click('.form-progress-buttons .usa-button-primary');
+      E2eHelpers.expectNavigateAwayFrom(client, '/personal-information/direct-deposit');
+
+      // Review and Submit Page.
+      client
+        .expect.element('label[name="privacyAgreement-label"]').to.be.visible;
+      // When you try to click on the label in the normal way, it'll instead click on the link
+      // inside the label that shows the popup. So we have to do this disgusting hack.
+      client.pause(1000);
+      client.execute((selector) => {
+        document.querySelector(selector).click();
+      }, ['label[name="privacyAgreement-label"]']);
```

Comment:
How odd...Is there a reason it has to be the label, or can you do the following?
```js
client.click('input[name="privacyAgreement"]');`
```